skiplang/prelude/ts/wasm/src/sk_types.ts

```diff
@@ -254,7 +254,14 @@ export class Utils {
     kind = kind ? kind : Stream.OUT;
     str += newLine ? "\n" : "";
     if (kind == Stream.DEBUG) {
-      this.stddebug.push(str);
+      const lines = str.split("\n");
+      let prefix = this.stddebug.join("");
+      this.stddebug = [];
+      for (let i = 0; i < lines.length - 1; i++) {
+        console.error(prefix + lines[i]!);
+        prefix = "";
+      }
+      this.stddebug.push(lines[lines.length - 1]!);
```

Comment:
```suggestion
      // Flush buffered this.stddebug output at newlines
      if ( str.indexOf("\n") == -1 ) this.stddebug.push(str)
      else {
        const idx = str.lastIndexOf("\n");
        console.error(this.stddebug.join("") + str.slice(0, idx));
        this.stddebug = [str.slice(idx + 1)];
      }
```

Same behavior but simpler/easier to understand, I think?